RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Some of my changes made a conflict in this file.  Please pull from Next and manually resolve this conflict, being sure to keep my updates.